bundle-workflow/python/build.py

```diff
@@ -3,31 +3,57 @@
 
 import os
 import sys
+import subprocess
 import tempfile
-import urllib.request
-from lib.build_manifest import BuildManifest
-from lib.build_output import BuildOutput
+import yaml
+from manifests.input_manifest import InputManifest
+from build_workflow.build_recorder import BuildRecorder
+from build_workflow.builder import Builder
+from build_workflow.git_repository import GitRepository
 
 if (len(sys.argv) < 2):
     print("Build an OpenSearch Bundle")
     print("usage: build.sh /path/to/manifest")
     exit(1)
 
-with tempfile.TemporaryDirectory() as work_dir:
-    manifest = BuildManifest.from_file(sys.argv[1])
-    build = manifest.build
-    output = BuildOutput()
+component_scripts_path = os.path.join(os.path.dirname(os.path.abspath(__file__)), '../scripts/bundle-build/components')
```

Comment:
If these are static paths it makes sense to me to build these paths directly inside of the Builder class.  Any reason these wouldn't be static?

Reply:
This code already exists in https://github.com/opensearch-project/opensearch-build/blob/2ac8a7a876f497d7870fc591dbf497f0606cd018/bundle-workflow/python/lib/component.py#L31

Reply:
This code should not be in `lib/component.py`. Setting up the default paths is not the responsibility of the builder (or the now-defunct `component`) - it only consumes those paths.

Also, the relative paths in `component.py` are super fragile. Moving the logic out into build.py means `builder` can rely on absolute paths (or, more accurately, `builder` doesn't care whether they are relative or absolute).